src/main/resources/assets/js/app/view/context/WidgetsSelectionRow.ts

```diff
@@ -118,3 +124,50 @@ export class WidgetViewOption {
     }
 
 }
+
+export class WidgetViewer extends NamesAndIconViewer<WidgetViewOption> {
+
+    constructor() {
+        super('widget-viewer');
+    }
+
+    doLayout(object: WidgetViewOption) {
+        super.doLayout(object);
+
+        const view = this.getNamesAndIconView();
+        if (object && object.getWidgetView() && view) {
+            const widgetClass = object.getWidgetView().getWidgetKey() != null ? 'external-widget' : 'internal-widget';
+            view.removeClass('external-widget internal-widget');
+            view.addClass(widgetClass);
+        }
+    }
+
+    resolveDisplayName(object: WidgetViewOption): string {
+        return object.getWidgetView().getWidgetName();
+    }
+
+    resolveSubName(object: WidgetViewOption): string {
+        const name = object.getWidgetView().getWidgetName();
+        const description = object.getWidgetView().getWidgetDescription() || 'No description';
+        switch (name) {
+        case 'Properties': return 'Properties of the content';
```

Comment:
Descriptions of the internal widgets must be localised, just like their names. Correct english descriptions:
1) _Content details_
2) _History of content changes_
3) _References to or from other items_